docs/package_templates/autotools_package/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package C) # if the project is pure C
```

since it's a CPP file in this example can we comment this line out? this way its more usable

```suggestion
#project(test_package C) # if the project is pure C
```

---

I just converted the autotools test_package to a C usage. Now we have a separated example, not only c++.